helm/dendrite/templates/deployment.yaml

```
@@ -0,0 +1,97 @@
+{{ template "validate.config" . }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: {{ $.Release.Namespace }}
+  name: {{ include "dendrite.fullname" . }}
+  labels:
+    app: {{ $.Chart.Name }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "dendrite.selectorLabels" . | nindent 6 }}
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        {{- include "dendrite.selectorLabels" . | nindent 8 }}
+    spec:
+      volumes:
+      - name: {{ .Release.Name }}-conf-vol
+        secret:
+          secretName: {{ .Release.Name }}-conf
+      - name: {{ .Release.Name }}-signing-key
+        secret:
+          secretName: {{ default (print .Release.Name "-signing-key") $.Values.signing_key.existingSecret | quote }}
+      {{- if (gt (len ($.Files.Glob "appservices/*")) 0) }}
+      - name:{{ .Release.Name }}-appservices
+        secret:
+          secretName: {{ .Release.Name }}-appservices-conf
+      {{- end }}
+      - name: {{ .Release.Name }}-jetstream
+        persistentVolumeClaim:
+          claimName: {{ default (print .Release.Name "-jetstream-pvc") $.Values.persistence.jetstream.existingClaim | quote }}
+      - name: {{ .Release.Name }}-media
```

please replace everywhere `.Release.Name` with your helper
```suggestion
      - name: {{ include "dendrite.fullname" . }}-media
```